game/BeepSaberMainMenu.gd

```diff
@@ -60,8 +60,8 @@ func _load_playlists():
 	
 	if (!_playlists):
 		vr.log_error("No songs found in " + bspath);
-		for b in $SongsMenu/Songs.get_children():
```

Comment:
Is this an intended change for this PR? What does it do?

Reply:
oh yea forgot abt that, it tried to clean the old list menu giving an error when there is no songs, i'll replace it by the clean function of the new itemlist that is used
so is just a little bugfix